src/Form/Type/RolesMatrixType.php

```diff
@@ -73,6 +75,7 @@ static function (Options $options, $value) {
 
                     return $value;
                 },
+            'exclude' => [UserInterface::ROLE_DEFAULT],
```

Comment:
```suggestion
            'excluded_roles' => [UserInterface::ROLE_DEFAULT],
```
to be more specific ?

Reply:
Also please add docs about this feature

Reply:
what is your opinions about always excluding `UserInterface::ROLE_DEFAULT` in addition to `excluded_roles` ?

Reply:
I would prefer it to be configurable, so no auto addition, wdyt @VincentLanglet ?

Reply:
I agree, seems better to let it always configurable